code/modules/paperwork/paper_premade.dm

```diff
@@ -31,6 +31,45 @@
 	name = "paper- 'Holopad Notice'"
 	default_raw_text = "<B>Can't get any botanists at the table? Have you tried using the damn holopad?</B><BR>\n<BR>\nStep onto the pad, and interface with it<BR>\nthen make your dang ol' call!<BR>\n<BR>\nYou want to call \"Hydroponics\" to reach them."
 
+/obj/item/paper/guides/jobs/engineering/bsa
+	name = "Bluespace Artillery (BSA) Assembly Instructions"
+	default_raw_text = {"<b>Below is a quick guide on setting up your new Bluespace Artillery Cannon.</b>
+	<br>
+	Featuring the newest in Nanotrasen technology, the Bluespace Artillery Cannon is your surefire ticket to complete and total threat elimination.
+	<br>
+	<h1>1. Acquire necessary resources.</h1>
+	* Bluespace Artillery Bore
+	* Bluespace Artillery Fusor
+	* Bluespace Artillery Generator
+	* Bluespace Artillery Controls
+	* 26 cable pieces
+	* 20 iron
+	* 20 bluespace crystals (or synthetic)
+	* 5 quadratic capacitors
+	* 5 femto servos
+	* 1 glass sheet
+	<h1>2. Find suitable location</h1>
+	The BSA has a steep power requirement and functions best when it is on its own APC.  Keep other items in the room to a minimum. Ensure the BSA is placed somewhere where it will not shoot at another part of the station.  Make sure the room isn't easily accessible by someone who shouldn't be there.
+	<h1>3. Construction</h1>
+	1. Build three machine frames.
+	2. Determine the desired direction you want your artillery to face.
+	3. Leave six spaces in front, four behind.  The artillery cannon finishes itself after the assemblies are aligned.
+	4. Place the machine frames in a horizontal line.
+	5. Wrench the three machine frames in place.
+	6. Install wires in the three machine frames.
+	7. Install the bore in the front machine frame. this is your barrel.
+	8. Install the fusor in the center machine frame.
+	9. Install the generator in the back machine frame.
+	10. Examine the three machines and see what else they need, you can use a screwdriver to finish the assembly when parts are installed.
+	11. Construct a computer terminal in front of the fusor, it must be directly south of it, wrench it to the ground and add the Controls board.
```

Comment:
I don't believe this is true, the console can be built in other locations. Which is often better as it can be in a secure location like the bridge.

Reply:
I'll give it a shot, I've only ever actually built the console in the same room as the BSA.  Also thank you for the corrections! I'll fix the grammatical errors and add in the information about the console.

Reply:
The console can be built in the same room as the BSA, they actually must exist in the same area but that is slightly technical and not necessary info for construction.  Trying to move the console to another room (even the bridge) makes it unable to detect the BSA assembly.

Reply:
If the CE wants to put it on the bridge they'll have to do some work themselves to make that happen, and they (hopefully) already know how to build a BSA.